.github/workflows/issue-comment.yml

```
@@ -23,6 +23,12 @@ jobs:
     env:
       GH_TOKEN: ${{ secrets.token }}
     steps:
+      - name: Checkout newrelic-php-agent code
+        uses: actions/checkout@v4
+        with:
+          path: php-agent
+          repository: ${{ inputs.origin }}/newrelic-php-agent
+          ref: ${{ inputs.ref }}
       - name: respond to issue
         run: |
           gh issue comment ${{ inputs.issue-number }} --body "${{ inputs.message }}"
```

I have not tried this but it should do the job without the checkout step:
```suggestion
          gh issue -R ${{ github.repository }} comment ${{ inputs.issue-number }} --body "${{ inputs.message }}"
```

---

Looks promising, let's give it a shot

---

changed here f96ac63b57cb2bb020c059c9873d402ad35456f7